ui-admin/src/populate/PopulateRouteSelect.tsx

```diff
@@ -4,28 +4,40 @@ import PopulatePortalControl from './PopulatePortalControl'
 import PopulateSurveyControl from './PopulateSurveyControl'
 import PopulateSiteContentControl from './PopulateSiteContent'
 import PopulateAdminConfig from './PopulateAdminConfig'
+import { navDivStyle, navListItemStyle } from 'util/subNavStyles'
+import { renderPageHeader } from 'util/pageUtils'
 
 /** shows links to the populate control panels, and handles the routing for them */
 export default function PopulateRouteSelect({ portalShortcode }: {portalShortcode?: string}) {
   /** styles links as bold if they are the current path */
-  const navStyleFunc = ({ isActive }: {isActive: boolean}) => {
-    return isActive ? { fontWeight: 'bold' } : {}
+  function getLinkCssClasses({ isActive }: { isActive: boolean }) {
+    return `${isActive ? 'fw-bold' : ''} d-flex align-items-center`
   }
```

Comment:
you can drop this function altogether and use `navLinkStyleFunc` from subNavStyles.ts below

Reply:
Done. do you want me to do this with EnrolleeView.tsx as well?